app/build.gradle

```diff
@@ -38,6 +38,15 @@ android {
     dexOptions {
         javaMaxHeapSize "4g"
     }
+    flavorDimensions "version"
+    productFlavors {
+        hrValidation {
+            dimension "version"
```

Comment:
I'd recommend adding a `versionNameSuffix` to avoid confusion and having different versionCodes (e.g. computed) if we want to allow deploying to the Play Store